cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -290,7 +301,26 @@ impl xcm_executor::Config for XcmConfig {
 	type SubscriptionService = PolkadotXcm;
 	type PalletInstancesInfo = AllPalletsWithSystem;
 	type MaxAssetsIntoHolding = MaxAssetsIntoHolding;
-	type FeeManager = XcmFeesToAccount<Self, WaivedLocations, AccountId, TreasuryAccount>;
+	type FeeManager = XcmFeeManagerFromComponents<
+		WaivedLocations,
+		(
+			XcmExportFeeToRelayerRewardAccounts<
```

Comment:
is this duplicated below? if so, pull out into a separate type to deduplicate.

Reply:
They are very similar, but they aren't duplicated. This one is for the AssetHubRococo -> AssetHubWococo bridge and the one below is for AssetHubWococo -> AssetHubRococo bridge.